packages/botonic-react/package.json

```diff
@@ -17,7 +17,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@botonic/core": "^0.10.1",
+    "@botonic/core": "0.11.0-alpha.0",
```

Comment:
in case of botonic/core, we didn't published it as it wasn't needed, but let's leave it as well as 0.11.0-alpha.1 to be consistent